com.microsoft.mrtk.graphicstools.unity/Samples~/MeshOutlines/Materials/leCat_mat.mat

```diff
@@ -0,0 +1,246 @@
+%YAML 1.1
```

Comment:
nit: The name of this material is in snake case. Would you mind changing to Pascal case like the other materials in this folder?